File: PDOO/NapakalakiRuby/lib/Prize.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

module NapakalakiGame
  class Prize
    attr_reader :treasures, :levels    #los consultores se generan de forma automatica. Los modificadores no. 

    def initialize (t,l)
     @treasures = t
     @levels    = l
    end

    public
    def to_s

      "\nTesoros: #{@treasures} \nNiveles: #{@levels}"

    end
  end
  
end

File: PDOO/NapakalakiJava/src/NapakalakiGame/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package NapakalakiGame;

/**
 *
 * @author pablo
 */
public class Prize {
    private int level;
    private int treasures;
    
    public Prize() {
        level = 0;
        treasures = 0;
    }
    public Prize(int level, int treasures) {
        this.level = level;
        this.treasures = treasures;
    }  
    
    public int getLevel(){
        return level;
    }
    
    public int getTreasures(){
        return treasures;
    }
    
    public String toString(){
        return "Treasures = " + Integer.toString(treasures) + " Levels = " + Integer.toString(level);
    }
}
